frontend/src/pages/storageClasses/StorageClassesTableRow.tsx

```diff
@@ -0,0 +1,204 @@
+    );
+
+    if (existingDefaultConfigMap) {
+      const [name, config] = existingDefaultConfigMap;
+      await updateStorageClassConfig(name, { ...config, isDefault: false });
+      refresh();
+    }
+  }, [storageClass.metadata.name, storageClassConfigMap, refresh]);
+
+  return (
+    <Tr>
+      <Td dataLabel={ColumnLabel.DisplayName}>
+        <TableRowTitleDescription
+          title={metadata.name}
+          description={storageClassConfig?.description}
+        />
+      </Td>
+
+      <Td dataLabel={ColumnLabel.OpenshiftStorageClass}>
+        <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+          <FlexItem>{storageClassConfig?.displayName}</FlexItem>
+
+          <FlexItem>
+            <Popover
+              position="right"
+              headerContent="Storage class info"
+              bodyContent={
+                <DescriptionList isCompact className="pf-v5-u-mt-lg">
+                  {storageClassInfoItems.map((storageClassInfoItem) => (
+                    <DescriptionListGroup key={storageClassInfoItem.label}>
+                      <DescriptionListTerm>{storageClassInfoItem.label}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {storageClassInfoItem.value}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  ))}
+                </DescriptionList>
+              }
+            >
+              <DashboardPopupIconButton
+                icon={<OutlinedQuestionCircleIcon />}
+                aria-label="Storage class info popover"
+              />
+            </Popover>
+          </FlexItem>
+
+          {isOpenshiftDefault && (
+            <FlexItem>
+              <Tooltip content="This is the default storage class in OpenShift.">
+                <Label color="green" isCompact data-testid="openshift-sc-default-label">
+                  Default
+                </Label>
+              </Tooltip>
+            </FlexItem>
+          )}
+        </Flex>
+      </Td>
+
+      <Td dataLabel={ColumnLabel.Enable}>
+        <StorageClassEnableSwitch
+          storageClassName={metadata.name}
+          isChecked={!!storageClassConfig?.isEnabled}
+          isDisabled={isEnableSwitchDisabled}
+          onChange={refresh}
+        />
+      </Td>
+
+      <Td dataLabel={ColumnLabel.Default}>
+        <StorageClassDefaultRadio
+          storageClassName={metadata.name}
+          isChecked={!!storageClassConfig?.isDefault}
+          isDisabled={!storageClassConfig?.isDefault && !storageClassConfig?.isEnabled}
+          onChange={onDefaultRadioChange}
+        />
+      </Td>
+
+      <Td dataLabel={ColumnLabel.LastModified}>
+        {storageClassConfig?.lastModified ? (
+          <Timestamp date={new Date(storageClassConfig.lastModified)} />
+        ) : (
+          'Unknown'
```

Comment:
use a dash `-`

Reply:
I can use a dash, but this approach is used here already:

https://github.com/opendatahub-io/odh-dashboard/blob/157e0b74f386f3859281b7f58d50231d54a572d4/frontend/src/pages/connectionTypes/ConnectionTypesTableRow.tsx#L112

Reply:
Ok then thats fine. It will be changed to this in another jira in the epic that covers the edge cases

https://www.figma.com/proto/372he4ynUVkovm3lwfFVbg/Storage-classes?page-id=773%3A20264&node-id=957-23329&node-type=FRAME&viewport=-1008%2C-3722%2C0.29&t=w46pKtLYGlSOkxQr-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=773%3A20265

so keeping this is good for now